source/pattern-library/content-views/list-view/index.md

```diff
@@ -0,0 +1,233 @@
+        <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="card-markup-2" href="#card-markup-2">Reference Markup</a></p>
+        <div class="collapse in" id="card-markup-2">
+          <pre class="prettyprint">{% capture markup_include %}{% include widgets/list-view/list-view-variation-1.html %}{% endcapture %}{{ markup_include | xml_escape }}</pre>
+        </div>
+        <h2 id="example-code-3">Row Variation #2</h2>
+        <div class="example-pf">
+          <div class="example-pf-demo example-pf-demo-no-padding">
+            <div class="cards-pf">
+              <div class="container-fluid container-cards-pf">
+                <div class="row row-cards-pf">
+                  {% include widgets/list-view/list-view-variation-2.html %}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="card-markup-3" href="#card-markup-3">Reference Markup</a></p>
+        <div class="collapse in" id="card-markup-3">
+          <pre class="prettyprint">{% capture markup_include %}{% include widgets/list-view/list-view-variation-2.html %}{% endcapture %}{{ markup_include | xml_escape }}</pre>
+        </div>
+        <h2 id="example-code-4">Row Variation #3</h2>
+        <div class="example-pf">
+          <div class="example-pf-demo example-pf-demo-no-padding">
+            <div class="cards-pf">
+              <div class="container-fluid container-cards-pf">
+                <div class="row row-cards-pf">
+                  {% include widgets/list-view/list-view-variation-3.html %}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="card-markup-4" href="#card-markup-4">Reference Markup</a></p>
+        <div class="collapse in" id="card-markup-4">
+          <pre class="prettyprint">{% capture markup_include %}{% include widgets/list-view/list-view-variation-3.html %}{% endcapture %}{{ markup_include | xml_escape }}</pre>
+        </div>
+        <h2 id="example-code-5">Row Variation #4</h2>
+        <div class="example-pf">
+          <div class="example-pf-demo example-pf-demo-no-padding">
+            <div class="cards-pf">
+              <div class="container-fluid container-cards-pf">
+                <div class="row row-cards-pf">
+                  {% include widgets/list-view/list-view-variation-4.html %}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="card-markup-5" href="#card-markup-5">Reference Markup</a></p>
+        <div class="collapse in" id="card-markup-5">
```

Comment:
Just a nit. Remove the "card-" prefix from "card-markup-5" and others -- there's no cards here. This id seems to be copied from other pages.